docs/HISTORY.txt

```diff
@@ -5,6 +5,8 @@ Changelog
 1.2.5 (unreleased)
 ------------------
 
+Add plone 5 default profile. [mathias.leimgruber]
```

Comment:
Please make it a list item by prefixing with `- `.

Reply:
@mbaechtold  done